### ISSUE TITLE
Feed the big list of naughty strings into the test suite

### ISSUE THREAD

**Issue by strugee** [pump-io/pump.io#1671]
https://github.com/minimaxir/big-list-of-naughty-strings/blob/master/blns.txt

**Comment by Serkan-devel** [comment]
Does it mean, I won't be able to type emotes in pump.io ?
uwu

**Comment by strugee** [comment]
No, the idea is to make sure that we handle these strings _correctly_ (i.e. that they do not cause any bugs, such as the string being mangled or dropped).

**Comment by Serkan-devel** [comment]
(｡◕ ∀ ◕｡)
